src/openforms/prefill/contrib/kvk/plugin.py

```diff
@@ -36,6 +36,9 @@ def get_available_attributes(self):
     def get_prefill_values(
         self, submission: Submission, attributes: List[str]
     ) -> Dict[str, Any]:
+        if not self.is_enabled:
+            raise PluginNotEnabled()
```

Comment:
Same here - I'd move this check to the code calling the method rather than checking it in the plugin itself.